### PR TITLE
MID :: Clust and Tracks normalisation + histo Digit Mean multi Hits

### PR DESCRIPTION
**Pull request by ValerieRamillien**

### CODE REVIEW

File: Modules/MUON/MID/include/MID/ClustQcCheck.h [hunk 1]
```diff
@@ -31,10 +31,18 @@ class ClustQcCheck : public o2::quality_control::checker::CheckInterface
...
   std::string getAcceptedType() override;
 
+ private:
+  float mClusterTF = 0;
+  int mOrbTF = 32;
+  // float scaleTime = 0.0114048; // 128 orb/TF * 3564 BC/orb * 25ns
+  float scaleTime = 0.0000891; // 3564 BC/orb * 25ns
+  float mClusterScale = 100;
+
   ClassDefOverride(ClustQcCheck, 2);
```
Comment (Barthelemy) [inline Modules/MUON/MID/include/MID/ClustQcCheck.h:46]: I am not sure, but should the class version be increased when the private members are added ?

**Review by Barthelemy — COMMENTED** [review]
thank you, this is good. 
My only concern is with the class version. I am not sure whether it should be increased or not when a private member is added.

File: Modules/MUON/MID/include/MID/ClustQcCheck.h (hunk 1 above)
Comment (Barthelemy) [reply Modules/MUON/MID/include/MID/ClustQcCheck.h:46]: I dived into ROOT's documentation and it says that it has to be increased: 
>  Every time you change the data members of a class, you should increase its ClassVersionID by one.

**Comment by ValerieRamillien** [comment]
OK thanks for the info, I will do it. 
Valerie. 


De: "Barthélémy von Haller" ***@***.***> 
À: "AliceO2Group/QualityControl" ***@***.***> 
Cc: "ValerieRamillien" ***@***.***>, "Author" ***@***.***> 
Envoyé: Jeudi 4 Mai 2023 13:31:52 
Objet: Re: [AliceO2Group/QualityControl] MID :: Clust and Tracks normalisation + histo Digit Mean multi Hits (PR #1755) 





@Barthelemy commented on this pull request. 


In [ https://github.com/AliceO2Group/QualityControl/pull/1755#discussion_r1184890994 | Modules/MUON/MID/include/MID/ClustQcCheck.h ] : 
>    Quality check(std::map<std::string, std::shared_ptr<MonitorObject>>* moMap) override;
   void beautify(std::shared_ptr<MonitorObject> mo, Quality checkResult = Quality::Null) override;
   std::string getAcceptedType() override;
 
+ private:
+  float mClusterTF = 0;
+  int mOrbTF = 32;
+  // float scaleTime = 0.0114048; // 128 orb/TF * 3564 BC/orb * 25ns
+  float scaleTime = 0.0000891; // 3564 BC/orb * 25ns
+  float mClusterScale = 100;
+
   ClassDefOverride(ClustQcCheck, 2); 


I dived into ROOT's documentation and it says that it has to be increased: 



Every time you change the data members of a class, you should increase its ClassVersionID by one.

**Review by Barthelemy — APPROVED** [review]

**Pull request merged**